Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,262 @@
+import SwiftUI
+
+/// Enables incremental changes of a counter without a direct input.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/stepper/)
+public struct Stepper: View {
+
+    @Binding var value: Int
+
+    let minValue: Int
+    let maxValue: Int
+    let isEnabled: Bool
+    let style: Style
+
+    @Environment(\.colorScheme) var colorScheme
+    
+    @State var isIncreased = false
+
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            decrementButton
+                .accessibility(addTraits: [.isButton])
```

Comment:
The `StepperButton` is probably already a button (if not, it should be).

We will also need to add specific accessibility identifiers (for UI tests) for all three parts (buttons and a value) so a call site can extract each part.

Reply:
Added specific identifiers